### DISCUSSION TITLE
Make wrangled tabs restorable or shareable across devices

### DISCUSSION THREAD

**Discussion by lmyers4371** [tabwrangler/tabwrangler#341] [General]
I had to restore my OS over the weekend and lost all my wrangled tabs.  It would be creat to have them available on all devices.

**Comment by ssorallen** [comment]
Chrome Extensions can only store 100KB in [sync'd storage](https://developer.chrome.com/docs/extensions/reference/api/storage#property-sync). The amount of data Tab Wrangler stores is too much for sync storage.

Perhaps in the future it could be sync'd with an external service, but this is not possible nor going to be addressed as is.
